answer_questions_using_vector_search_weaviate/main.bal

```diff
@@ -0,0 +1,52 @@
+import ballerina/http;
+import ballerinax/openai.embeddings;
+import ballerinax/weaviate;
+
+configurable string openAIToken = ?;
+configurable string weaviateToken = ?;
+configurable string weaviateURL = ?;
+
+const CLASS_NAME = "QuestionAnswerStore";
+
+final embeddings:Client openai = check new ({auth: {token: openAIToken}});
+final weaviate:Client weaviate = check new ({auth: {token: weaviateToken}}, weaviateURL);
+
+service / on new http:Listener(8080) {
+    resource function get answer(string question) returns error|record {|weaviate:JsonObject...;|}? {
+
+        // Retrieve OpenAI embeddings for the input question
+        embeddings:CreateEmbeddingResponse embeddingResponse = check openai->/embeddings.post({
+                model: "text-embedding-ada-002",
```

Comment:
Can we make `text-embedding-ada-002` a constant?

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/60/commits/b266f95fb9a512010774915db533652910a42502. However, in earlier samples we did not define model as a constant.